lnwire/error.go

```diff
@@ -32,6 +32,11 @@ const (
 	// FundingOpen request for a channel that is above their current
 	// soft-limit.
 	ErrChanTooLarge ErrorCode = 3
+
+	// ErrNonZeroPushAmount is return by a remote peer that receives a
+	// FundingOpen request for a channel with non-zero push amount while
+	// they have 'rejectpush' enabled.
+	ErrNonZeroPushAmount ErrorCode = 4
```

Comment:
These error codes are actually deprecated iirc. Instead you should define the error message here: https://github.com/lightningnetwork/lnd/blob/68d96488cc9f78e6aeb7d5c27822f384f756f335/lnwallet/errors.go#L16

Reply:
Moved it as requested.